src/views/migrationpolicies/list/components/MigrationPolicyMatchLabelSelectorList/MigrationPolicyMatchLabelSelectorLabel.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Label, Popover, PopoverPosition } from '@patternfly/react-core';
+
+import '../MigrationPolicySelectorLabel.scss';
+
+type MigrationPolicyMatchLabelSelectorLabelProps = {
+  matchKey: string;
+  value: string;
+  isVMILabel?: boolean;
+};
+
+export const MigrationPolicyMatchLabelSelectorLabel: React.FC<MigrationPolicyMatchLabelSelectorLabelProps> =
+  React.memo(({ matchKey, value, isVMILabel }) => {
+    const labelBodyContent = `${matchKey}=${value}`;
+    return (
+      <Popover
+        position={PopoverPosition.top}
+        aria-label="Match label selector"
+        bodyContent={`${matchKey}: ${value}`}
+        headerContent={'Match label'}
+      >
+        <Label
+          color={isVMILabel ? 'grey' : 'blue'}
```

Comment:
I can see isVMILabel appear in a few files and passed down. Can be this be managed by last component it self?

Reply:
we can't identify which selector was selected (VMI or Namespace) by the selector since they're the same